Cargo.toml

```
@@ -299,7 +304,9 @@ base64 = '0.22'
 bincode = '1'
 bit_field = '0.10'
 bitvec = '1'
-bls12_381 = { git = 'https://github.com/grandinetech/universal-precompiles.git', tag = 'bls12_381-6bb9695-up.0' }
+# bls12_381 = { git = 'https://github.com/grandinetech/universal-precompiles.git', tag = 'bls12_381-6bb9695-up.0' }
+bls12_381 = { git = "https://github.com/jimmychu0807/grandine-universal-precompiles.git", branch = "zkcrypto/bls12_381" }
```

Once https://github.com/grandinetech/universal-precompiles/pull/3 is merged, modify to

```suggestion
bls12_381 = { git = "https://github.com/grandine/universal-precompiles.git", branch = "zkcrypto/bls12_381" }
```